docs/CONFIG-VARS.md

```diff
@@ -46,6 +46,7 @@ You can use `default_public_access_cidrs` to set a default range for all created
 
 | Name | Description | Type | Default | Notes |
 | :--- | ---: | ---: | ---: | ---: |
+| use_private_endpoint | Creates a cluster api endpoint with a private ip address instead of a public ip address | bool | | When "true", all "*_public_access_cidrs" options may not use public ip addresses. Access to the cluster is only allowed from restricted IP ranges. |
```

Comment:
I would say given we have 3 providers and would need a flag for `private_cluster` support, this flag should be more generic. This is specific to GCP. I believe on the AWS side I'll be using `private_cluster` as the flag to tell the system to disable any `public` facing networking, etc.